portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.testcase

```diff
@@ -0,0 +1,22 @@
+@component-name = "portal"
+definition {
+
+	property testray.main.component.name = "Navigation";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	@description = "Access Portal Global Menu and validate CONFIGURATION string at Control Panel"
+	@priority = 5
```

Comment:
Hey, remember this past PR https://github.com/manuferreira/liferay-portal/pull/72/files? I asked you to change some stuff related to the description of the text, can you take a look again and try to apply it to this PR?

Reply:
Fixed. Thanks!